src/store/index.ts

```diff
@@ -0,0 +1,3 @@
+export { useSelector, useDispatch } from 'react-redux';
```

Comment:
These hooks aren't type-safe by default, but you can use a simple wrapper function.

Example: https://github.com/MyCryptoHQ/signer/blob/master/src/app/store/utils/useSelector.ts

Reply:
Ah great! I updated `src/store/index.tsx` to export the type-safe version.